diff --git bazel/repository_locations.bzl bazel/repository_locations.bzl
index a262e3fe44..5c95a0ef6c 100644
--- bazel/repository_locations.bzl
+++ bazel/repository_locations.bzl
@@ -1100,7 +1100,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_url = "https://quiche.googlesource.com/googleurl",
         # Static snapshot of https://quiche.googlesource.com/googleurl/+archive/dd4080fec0b443296c0ed0036e1e776df8813aa7.tar.gz
         version = "dd4080fec0b443296c0ed0036e1e776df8813aa7",
-        sha256 = "59f14d4fb373083b9dc8d389f16bbb817b5f936d1d436aa67e16eb6936028a51",
+        sha256 = "fc694942e8a7491dcc1dde1bddf48a31370a1f46fef862bc17acf07c34dc6325",
         urls = ["https://storage.googleapis.com/quiche-envoy-integration/{version}.tar.gz"],
         use_category = ["controlplane", "dataplane_core"],
         extensions = [],
