diff --git bazel/repository_locations.bzl bazel/repository_locations.bzl
index 27f6fd0bfc..056784eeef 100644
--- bazel/repository_locations.bzl
+++ bazel/repository_locations.bzl
@@ -426,7 +426,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "Intel® Integrated Performance Primitives Cryptography",
         project_url = "https://github.com/intel/ipp-crypto",
         version = "2021.6",
-        sha256 = "632cc5ba54413eeab575682619c05d247e9b7f2fc58ea3e5f4a02bdcab3e6b78",
+        sha256 = "a52bf15208d493adb846994f2ce928bd02c74fd8ff3a2def2fca7b072d67e6bf",
-        strip_prefix = "ipp-crypto-ippcp_{version}",
+        strip_prefix = "cryptography-primitives-ippcp_{version}",
         urls = ["https://github.com/intel/ipp-crypto/archive/ippcp_{version}.tar.gz"],
         release_date = "2022-08-09",
